### PR TITLE
Fix `Pluck` example #439

### LINKED ISSUE #439: [Bug]: select usage query multiple columns is wrong
## Document Link

[advanced_query.html#Pluck](https://gorm.io/docs/advanced_query.html#Pluck)

db.Select("name", "age").Scan(&users)
db.Select("name", "age").Find(&users)

There is a mistake, query multiple columns correct usage is:
> db.Select([]string{"name", "age"}).Scan(&users)

[chainable_api.go#L81](https://github.com/go-gorm/gorm/blob/master/chainable_api.go#L81)

### PR DESCRIPTION
**Pull request by s-takehana**
- [x] **ONLY** change English documents in the Pull Request, translations will be synced and translate them with https://translate.gorm.io/ or it will cause merge conflicts!

### What did this pull request do?

Fix #439

### CODE REVIEW

**Review by a631807682 — CHANGES_REQUESTED** [review]
This problem not only in here, but also in the following document.
```
pages/docs/create.md
pages/docs/hooks.md
pages/docs/performance.md
pages/docs/query.md
pages/docs/sql_builder.md
pages/docs/update.md
pages/docs/v2_release_note.md
```

**Comment by s-takehana** [comment]
@a631807682 Thank you for information. Let me confirm one thing.
Do you mean `Select([]string{"name", "age"})` is much better than `Select("name", "age")` ?
It seems both are fine.

GORM version is `v1.21.12` .
```go
package main

import (
	"fmt"
	"gorm.io/driver/postgres"
	"gorm.io/gorm"
)

type User struct {
	gorm.Model
	Name string
	Age  uint
}

func main() {
	dsn := "host=postgres user=postgres password=***** dbname=postgres port=5432"
	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
	if err != nil {
		panic("failed to connect database")
	}

	db.AutoMigrate(&User{})

	db.Create(&[]User{
		{Name: "User1", Age: 20},
		{Name: "User2", Age: 20},
	})

	var users []User

	db.Debug().Model(&User{}).Select("name", "age").Scan(&users)
	fmt.Printf("%+v\n", users)

	db.Debug().Table("users").Select([]string{"name", "age"}).Scan(&users)
	fmt.Printf("%+v\n", users)

	db.Debug().Select("name", "age").Find(&users)
	fmt.Printf("%+v\n", users)

	db.Debug().Select([]string{"name", "age"}).Find(&users)
	fmt.Printf("%+v\n", users)
}
```

```
2021/08/01 08:46:05 /workspace/main.go:31
[0.794ms] [rows:2] SELECT "name","age" FROM "users" WHERE "users"."deleted_at" IS NULL
[{Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User1 Age:20} {Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User2 Age:20}]

2021/08/01 08:46:05 /workspace/main.go:34
[0.612ms] [rows:2] SELECT name,age FROM "users"
[{Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User1 Age:20} {Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User2 Age:20}]

2021/08/01 08:46:05 /workspace/main.go:37
[0.351ms] [rows:2] SELECT "name","age" FROM "users" WHERE "users"."deleted_at" IS NULL
[{Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User1 Age:20} {Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User2 Age:20}]

2021/08/01 08:46:05 /workspace/main.go:40
[0.344ms] [rows:2] SELECT "name","age" FROM "users" WHERE "users"."deleted_at" IS NULL
[{Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User1 Age:20} {Model:{ID:0 CreatedAt:0001-01-01 00:00:00 +0000 UTC UpdatedAt:0001-01-01 00:00:00 +0000 UTC DeletedAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false}} Name:User2 Age:20}]
```

**Comment by a631807682** [comment]
It seems I used an old version, it generate sql like this, i have just tested the code above is no problem.
```
 SELECT `name` = "age" FROM "users"
```

**Comment by s-takehana** [comment]
@a631807682 I see.

File: pages/docs/advanced_query.md [hunk 1]
```diff
@@ -321,8 +321,8 @@ db.Model(&User{}).Distinct().Pluck("Name", &names)
 // SELECT DISTINCT `name` FROM `users`
 
 // Requesting more than one column, use `Scan` or `Find` like this:
-db.Select("name", "age").Scan(&users)
-db.Select("name", "age").Find(&users)
+db.Model(&User{}).Select([]string{"name", "age"}).Scan(&users)
```
Comment (s-takehana) [inline pages/docs/advanced_query.md:324]: `Scan` requires `Model` or `Table`.

**Comment by jinzhu** [comment]
The old style should works?

**Pull request closed without merging**